.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+require('@bigcommerce/eslint-config/patch');
```

Comment:
IIRC this is really for monorepos and our internal microapps using our ng-build. I don't think we need this here.

Reply:
@chanceaclark Removing this require causes ESLint to throw `ESLint couldn't find the plugin "@typescript-eslint/eslint-plugin". (The package "@typescript-eslint/eslint-plugin" was not found when loaded as a Node module from the directory "~/.../sample-app-nodejs".)` then prompts to install `@typescript-eslint/eslint-plugin`. Seems like we need one or the other.